client/packages/common/src/intl/locales/en/patients.json

```diff
@@ -18,6 +18,7 @@
   "label.visit-date": "Visit Date",
   "label.visit-start": "Start",
   "label.visit-end": "End",
+  "label.seen-by": "Seen by",
```

Comment:
"Seen by"? or should it just be "Clinician"?

Reply:
good question @richardmoizeau  ? "Clinician" sound clearer?

Reply:
Yes, `Clinician` is better than `Seen by`

Reply:
👍 "Clinician" is is! f191ed3e7